docs/content/Tutorial:-A-First-Look-at-Druid.md

```diff
@@ -72,13 +78,13 @@ You should see a bunch of files:
 Running Example Scripts
 -----------------------
 
-Let's start doing stuff. You can start a Druid [Realtime](Realtime.html) node by issuing:
+Let's start doing stuff. You can start an example Druid [Realtime](Realtime.html) node by issuing:
 
 ```
 ./run_example_server.sh
 ```
 
-Select "wikipedia".
+Select "2" for the "wikipedia" example.
```

Comment:
is it possible to support both selecting a number and typing it in?

The reason I ask is I always run with ./run_example_server.sh wikipedia